src/store/observations.js

```diff
@@ -4,54 +4,50 @@ import Vue from 'vue'
 export default {
   namespaced: true,
   state: {
-    data: {
-      123: {
-        id: 123,
-        locationId: 123
-      },
-      456: {
-        id: 123,
-        locationId: 123,
-        subject: 'Pamela'
-      },
-      789: {
-        id: 123,
-        locationId: 123,
-        subject: 'Pamela',
-        actionName: 'walking',
-        modifierName: 'quickly'
-      }
-    }
+    data: {}
```

Comment:
The `Observations` view depends on this for rendering so I think this needs to be null as default.